src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -157,9 +157,10 @@ public ContentWriter getWriter(NodeRef nodeRef, QName propertyQName, boolean upd
      * Gets a presigned URL to directly access a binary content. It is up to the content store
      * if it can fulfil this request with an expiry time or not.
      *
-     * @param contentUrl A content store URL
+     * @param nodeRef a reference to a node having a content property
      * @param expiryTime Expiration time in milliseconds 
```

Comment:
I think we need to be a little clearer with the explanation of the expiryTime parameter. Is it the time in epoch seconds when the link expires or the length of time in milliseconds that the link is valid for?

Reply:
The length of time in milliseconds that the link is valid for. I will update the description.